docs/application-connector/docs/040-remote-evironment-custom-resource.md

```diff
@@ -0,0 +1,75 @@
+---
+title: RemoteEnvironment
+type: Custom Resource
+---
+
+The `remoteenvironments.remoteenvironment.kyma.cx` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to register a Remote Environment in Kyma. The RemoteEnvironment resource defines APIs that the Remote Environment offers. As a result, the RemoteEnvironment is mapped to ServiceClasses in the Service Catalog. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd remoteenvironments.remoteenvironment.kyma.cx -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource that registers the `re-prod` Remote Environment which provides one service with the `ac031e8c-9aa4-4cb7-8999-0d358726ffaa` ID.
+
+```
+apiVersion: remoteenvironment.kyma.cx/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  name: re-prod
+spec:
+  source:
+    environment: "production"
+    type: "commerce"
+    namespace: "com.github"
+  description: "RE description"
+  accessLabel: "re-access-label"
+  services:
+    - id: "ac031e8c-9aa4-4cb7-8999-0d358726ffaa"
+
+      displayName: "Promotions"
+      longDescription: "Promotions APIs"
+      providerDisplayName: "Organization name"
+
+      tags:
+      - occ
+      - Promotions
+
+      entries:
+      - type: API
+        gatewayUrl: "http://promotions-gateway.production.svc.cluster.local"
+        accessLabel: "access-label-1"
+        targetUrl: "http://10.0.0.54:9932/occ/promotions"
+        oauthUrl: "http://10.0.0.55:10219/occ/token"
+        credentialsSecretName: "re-ac031e8c-9aa4-4cb7-8999-0d358726ffaa"
+      - type: Events
+```
+
+## Custom resource parameters
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Mandatory?      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** |    **YES**   | Specifies the name of the CR. |
+| **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
+| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
+| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
+| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
+| **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
+| **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-environment-mapping-custom-resource.md) is created. |
+| **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
+| **spec.services.id** |    **YES**   | Identifies the service provided by the Remote Environment. |
```

Comment:
Similarly as above? ` Identifies the service that the Remote Environment provides.`